module/extension/Director/src/director/run_task_pack.cpp

```diff
@@ -268,18 +338,19 @@ namespace module::extension {
             }
         }
 
-        // This new task pack we are trying to run might be different from the task pack previously ran.
-        // There might be some tasks that weren't emitted in the new task pack.
-        // These tasks need to be removed from the places they were running.
         for (const auto& t : group.subtasks) {
```

Comment:
Probably should put a comment here as to why it's being removed after (to stop the single frame run)